engine/src/lib.rs

```diff
@@ -42,7 +42,15 @@ pub struct ServalEngine {
 impl ServalEngine {
     /// Create a new serval engine.
     pub fn new(extensions: HashMap<String, ServalExtension>) -> Result<Self, ServalEngineError> {
-        let engine = Engine::default();
+        let mut config = Config::default();
+        config.cache_config_load_default().map_err(|_| {
```

Comment:
Worried about the way wasmtime just intransparently loads some pre-cached binary... Nothing to change here, but I made #51 to follow up on it